.github/workflows/sync-end-to-end.yml

```diff
@@ -0,0 +1,64 @@
+name: Sync-End-to-End tests
+
+on:
+  push
```

Comment:
This, as I understand, will be removed before merging and we'll keep it running on schedule (as per the code below that's currently commented out?)

Reply:
Oh, yes! Sorry I didn't mention in the description 🤦‍♂️

Reply:
Updated now!